<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>member_input_dialog</class>
 <widget class="QDialog" name="member_input_dialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>391</width>
    <height>106</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>맴버 추가</string>
  </property>
  <widget class="QLineEdit" name="name_lineEdit">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>50</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="gender_comboBox">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>50</y>
     <width>76</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>남</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>여</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="name_label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>56</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>이름</string>
   </property>
  </widget>
  <widget class="QLabel" name="gender_label">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>30</y>
     <width>56</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>성별</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ok_button">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>50</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>확인</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
